lib/Paws/Net/Caller.pm

```diff
@@ -29,7 +29,7 @@ package Paws::Net::Caller;
         (defined $requestObj->content)?(content => $requestObj->content):(),
       }
     );
-    return ($response->{status}, $response->{content}, $response->{headers});
+    return $self->caller_to_response($service, $call_object, $response->{status}, $response->{content}, $response->{headers});
```

Comment:
That's fine with me. It's not something I changed but rather something that was introduced following a pull and then I had to tweak it to get it working again after it broke my scripts